lib/stack_master/parameter_resolvers/latest_ami.rb

```diff
@@ -0,0 +1,19 @@
+module StackMaster
+  module ParameterResolvers
+    class LatestAmi < Resolver
+      array_resolver class_name: 'LatestAmis'
+
+      def initialize(config, stack_definition)
+        @config = config
+        @stack_definition = stack_definition
+        @ami_finder = AmiFinder.new(@stack_definition.region)
+      end
+
+      def resolve(value)
+        owners = Array(value.fetch('owners', 'self').to_s)
+        filters = @ami_finder.build_filters_from_hash(value['filters'])
```

Comment:
It'd be better to use `value.fetch('filters')` so that a more obvious error is raised when filters is missing.